src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.model;
+
+import java.util.Objects;
+
+public class Fruit {
+    private String name;
+
+    public Fruit(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }

```

Comment:
You don't use it anywhere. Remove set() 
All the more, since Fruit is the key of HashMap, let us make it closer to immutable))